flink-flux-core/src/main/java/com/uber/athena/flux/flink/compiler/impl/datastream/FluxContext.java

```diff
@@ -16,8 +16,9 @@
  * limitations under the License.
  */
 
-package com.uber.athena.flux.flink.compiler;
+package com.uber.athena.flux.flink.compiler.impl.datastream;
 
+import com.uber.athena.flux.flink.compiler.api.CompilerVertex;
```

Comment:
`FluxContext` should be moved to top level API since it should not link with any compiler specific. this context belongs to flux.